test/test_fx.py

```diff
@@ -524,10 +525,25 @@ def test_graph_fns(self):
         mod.linear = torch.nn.Linear(3, 4)
         mod.bias = torch.rand(4)
         gm = GraphModule(mod, g)
-        input = torch.rand(3) 
+        input = torch.rand(3)
         r = gm(input)
         ref = torch.sin(mod.linear(input) + mod.bias)
-        self.assertEqual(r, ref) 
+        self.assertEqual(r, ref)
+
+    def test_replace_all_uses_with(self):
+        class testModule(torch.nn.Module):
+            def forward(self, a, b):
+                return a + b
+
+        m = testModule()
+        traced = symbolic_trace(m)
+        input1 = torch.randn(1)
+        input2 = torch.randn(1)
+        assert (input1 + input2) == traced(input1, input2)
+        GraphManipulation.replace_all_uses_with(
+            traced, "call_function", operator.add, "call_function", operator.mul
```

Comment:
Nit: using `kwargs` at the callsite can make it easier for the reader to understand what's going on, e.g. `replace_all_uses_with(fx_module=traced, old_op="call_function", old_target=operator.add, new_op="call_function", new_target=operator.mul)`